libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -27,6 +27,9 @@
 
 #if AP_AHRS_NAVEKF_AVAILABLE
 
+#define ATTITUDE_CHECK_THRESH_ROLL_PITCH_RAD radians(5)
```

Comment:
Could pre-square these.

Reply:
yes, indeed.. i think because it's so rarely called and only when disarmed it's probably not worth the optimisation.  txs though